src/components/templates/ScreeningRoom/useScreeningRoom.ts

```diff
@@ -0,0 +1,148 @@
+import { useState, useMemo, useCallback } from "react";
```

Comment:
Is this file/hook intentionally in `/ScreeningRoom/`

Reply:
Yeah, not sure if this is the best approach, but it's only used here, so I decided there was no sense to move it into hooks/

Reply:
I'm fine with this pattern for 'local dependencies', we follow it in a few places already, and it aligns closer with (what I feel is) a superior pattern of 'code layout', which is 'feature focussed' rather than 'type of thing focussed'.